theme/themes/pastanaga/elements/input.overrides

```diff
@@ -4,6 +4,7 @@
 
 .ui.input input {
   font-weight: @inputFontWeight;
+
```

Comment:
@lkttyagi Also, please configure your IDE to use eslint and prettier correctly. There're some unnecessary spaces here.